pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -467,6 +471,34 @@ func (az *Cloud) ensurePublicIPExists(service *v1.Service, pipName string, domai
 	return &pip, nil
 }
 
+func getIdleTimeout(s *v1.Service) (*int32, error) {
+	const (
+		min = 4
+		max = 30
+	)
+	errInvalidTimeout := fmt.Errorf("idle timeout value must be a whole number representing minutes between %d and %d", min, max)
+
+	val, ok := s.Annotations[ServiceAnnotationLoadBalancerIdleTimeout]
+	if !ok {
+		// Return a nil here as this will set the value to the azure default
+		return nil, nil
+	}
+	if val == "" {
```

Comment:
is it necessary to check here since below code would still parse `val`:
```
to, err := strconv.Atoi(val)
```

Reply:
Not necessary, removing this explicit check.